### PR TITLE
Add release notes for version-0.37.0

### PR DESCRIPTION
**Pull request by MostafaMegahid**
Closes

**What changed?**
Added release notes for enterprise version 0.37.0

**Why was this change made?**
update the release notes in the public docs.

**How was this change implemented?**
changed the docs.

**How did you validate the change?**
Reviewed it locally w/ yarn

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Review by enekofb — APPROVED** [review]

File: website/docs/enterprise/getting-started/releases-enterprise.mdx [hunk 1]
```diff
@@ -10,6 +10,44 @@ import TierLabel from "../../_components/TierLabel";
...
+- Added new tab for events in the Cluster Discovery view
+- Added a new component for metadata in the Automated Cluster Discovery view
+- Enhance filter headings style for better readability
+
+### Bug Fixes
+- Support dark mode on Explorer pagination controls
+- Fixes AutomatedClusterDiscovery deletion
+- Fix cluster creation URL bug with explorer
+- Bootstrap CLI: Added validation case for repo url without scheme
+- Add Yaml tab to Pipeline details
```
Comment (enekofb) [inline website/docs/enterprise/getting-started/releases-enterprise.mdx:40]: I could see the enhancements and bug fixes section. we dont usually have them  as stated in [here](https://github.com/weaveworks/weave-gitops-enterprise/blob/main/docs/releasing.md?plain=1#L94) ... see also past releases that only have `highlights`, `breaking changes`, etc ... 

(Btw, I see there is a case for bringing bug fixes to the release notes as our ee users would benefit from but raising the point as the release notes for this version wont be consistent with previous versions)

Comment (MostafaMegahid) [reply website/docs/enterprise/getting-started/releases-enterprise.mdx:40]: I've removed any technical enhancements or bug fixes from the original release notes, just including relevant user updates. I belive both enhancements and bug fixes should be included moving forward.

Comment (enekofb) [reply website/docs/enterprise/getting-started/releases-enterprise.mdx:40]: 👍  +1 on it, i guess that then we could reflect that expectation in the [releasing doc](https://github.com/weaveworks/weave-gitops-enterprise/blob/main/docs/releasing.md) where suits. 

(Btw, feel free to extend this one  https://github.com/weaveworks/weave-gitops-enterprise/pull/3677/files  which is pending of review)

Comment (MostafaMegahid) [reply website/docs/enterprise/getting-started/releases-enterprise.mdx:40]: added an edit to the process PR. https://github.com/weaveworks/weave-gitops-enterprise/pull/3677/files#r1409134334

**Pull request merged**